sync_acellera_conda_channel_deps.py

```diff
@@ -30,14 +31,22 @@
     ['conda-forge', 'pyemma'],
     ['conda-forge', 'thermotools'],
     ['bioconda', 'nglview'],
-    ['omnia', 'nglview'],  # Adding both omnia and bioconda nglview because they provide different builds
     ['rdkit', 'rdkit'],
     ['rdkit', 'boost'],
     ['conda-forge', 'nlopt'],
     ['conda-forge', 'periodictable']
 ]
 
+# Package arguments
+
+allpackages = [i[1] for i in packages]
```

Comment:
Mmm, it's a style thingy, sorry if being pedantic but use `i` for loop indexes only. Use `p` or `package` here better.

Reply:
totally agree. I was going to use package, but it's really a list with channel and package. I'll use p.